system/CodeIgniter.php

```diff
@@ -407,6 +407,11 @@ protected function handleRequest(RouteCollectionInterface $routes = null, Cache
 			{
 				return $returnResponse ? $possibleResponse : $possibleResponse->pretend($this->useSafeOutput)->send();
 			}
+
+			if ($possibleResponse instanceof RequestInterface)
```

Comment:
Change this to this will solve two of phpstan errors.
```suggestion
			if ($possibleResponse instanceof Request)
```